src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/ContentEncryptionAlgorithmTests.cs

```diff
@@ -94,16 +94,14 @@ public static void DecodeAlgorithmRc2_40_RoundTrip()
             }
 
             byte[] encodedMessage = ecms.Encode();
-
-
+            
```

Comment:
On the one hand, two blank lines became one.  Yay.

On the other, 2/4 characters have now become 9/10, because of all the encoded spaces.  Boo.

Tidy, please :)